geoadmin/README.md

```diff
@@ -0,0 +1,31 @@
+# STAC Browser for geo.admin.ch
+
+This is a fork from [radiantearth/stac-browser](https://github.com/radiantearth/stac-browser) containing some
+specific scripts and feature to deploy the STAC browser on [data.geo.admin.ch/browser/](https://data.geo.admin.ch/browser/).
+
+- [GIT Branch Strategy](#git-branch-strategy)
+- [Deploy](#deploy)
+
+## GIT Branch Strategy
+
+In order to simplify the merge from the fork, we are not allowed to do any changes directly into the `master` branch,
+this branch should be kept up to date with the fork master branch.
+
+Any geoadmin changes should be done in `geoadmin_master` branch. This branch follows the general geoadmin GIT FLOW rules.
+To get the lastest changes from the radiantearth, we first merge radiantearth master branch into our master branch and
```

Comment:
We might not be able to adopt any upstream change right away. We'll probably stay on the current version until version 3 is release... maybe make a comment that new upstream versions should be thoroughly tested before merged into geoadmin_master

Reply:
done